src/interfaces/package-export-transport.ts

```diff
@@ -48,18 +48,16 @@ export interface PackageKeyAndVersionPair {
 export interface NodeExportTransport {
```

Comment:
Should we also add the `PackageNodeExportTransport`?

Reply:
There is no benefit in doing this